salt/osfamilymap.yaml

```diff
@@ -13,7 +13,8 @@
 {%- set osmajorrelease = salt['grains.get']('osmajorrelease', osrelease)|string %}
 {%- set oscodename = salt['grains.get']('oscodename') %}
```

Comment:
If the variables `osmajorrelease` and `oscodename` are no longer used they should be removed.

Reply:
Done removed all unused variables